src/content/lodash/object/merge/spec.js

```diff
@@ -0,0 +1,93 @@
+const lodash = require('./lodash')
```

Comment:
this import is wrong, it should be `require('lodash')` for the tests to run.
once this is fixed the tests fail tho, I hope it's not due to the changes I asked you